.circleci/config.yml

```diff
@@ -329,14 +307,20 @@ commands:
           command: |
             if [[ "<< parameters.project >>" == "production" ]]; then
               export FEATURE_FLAGS_SHOW_RECOVERY_KEY_V2=true
+              export GREP="--grep=\"severity-1\""
+            elif [[ "<< parameters.project >>" == "stage" ]]; then
+              export FEATURE_FLAGS_SHOW_RECOVERY_KEY_V2=true
+              export GREP="--grep=\"severity-(1|2)\""
             else
               export FEATURE_FLAGS_SHOW_RECOVERY_KEY_V2=true
+              GREP=""
```

Comment:
Nit: We should keep it consistent. If we use export above, let's use export here. I think the exports are optional though. So either remove them, or use them.